### PR TITLE
Show thread messages <agile>

### PR DESCRIPTION
**Pull request by Devansh-bit**
@chirag-ghosh Please work you front end magic and show the replies somehow

### CODE REVIEW

**Comment by Devansh-bit** [comment]
@harshkhandeparkar pls review and merge

File: excretor/templates/thread.html [hunk 1]
```diff
@@ -1 +1,28 @@
-<!-- templates/thread.html -->
+TODO
```
Comment (harshkhandeparkar) [inline excretor/templates/thread.html:1]: If this is not used currently, just remove the file and do it in a separate pr.

File: excretor/src/dbmodels.rs [hunk 2]
```diff
@@ -16,6 +16,25 @@ pub struct DBMessageAndUser {
     pub email: String,
     pub deleted: bool,
     pub is_bot: bool,
+    pub cnt: Option<i64>,
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct DBReply {
```
Comment (harshkhandeparkar) [inline excretor/src/dbmodels.rs:23]: Why is reply a different struct? Can it not just be a `Message` struct?

File: excretor/src/models.rs [hunk 3]
```diff
@@ -50,6 +73,18 @@ impl User {
             is_bot: db_message_and_user.is_bot,
         }
     }
+    pub fn from_db_reply(db_reply: &dbmodels::DBReply) -> Self {
```
Comment (harshkhandeparkar) [inline excretor/src/models.rs:76]: Can this be implemented using the `From<T>` trait?

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

File: excretor/src/models.rs (hunk 3 above)
Comment (Devansh-bit) [reply excretor/src/models.rs]: Donee

**Comment by Devansh-bit** [comment]
Closes #32

**Review by harshkhandeparkar — APPROVED** [review]
amazing
